tap_hubspot/hubspot.py

```diff
@@ -197,20 +137,17 @@ def get_contacts(
             properties,
         )
 
-        for chunk in chunker(gen, 10):
+        for chunk in chunker(gen, 100):
```

Comment:
split this into two commits - removing the old configurations stuff is low-risk stuff, while this is important I wouldn't want this getting lost in the PRs.